cacher/cacher.go

```diff
@@ -53,28 +52,25 @@ func (cacher *Cacher) Handle(_ phi.Task, message phi.Message) {
 	if !ok {
 		cacher.logger.Panicf("[cacher] unexpected message type %T", message)
 	}
-	params, err := msg.Request.Params.MarshalJSON()
+
+	paramsBytes, err := json.Marshal(msg.Params)
 	if err != nil {
 		cacher.logger.Errorf("[cacher] cannot marshal request to json: %v", err)
 		msg.RespondWithErr(jsonrpc.ErrorCodeInternal, err)
```

Comment:
Maybe we should change this from `jsonrpc.ErrorCodeInternal` to `jsonrpc.ErrorCodeInvalidParams`.